RepulsiveField body
RepulsiveField left_hand
RepulsiveField right_hand